0300-method.Rmd

```diff
@@ -26,7 +26,7 @@ To identify priorities for crossing structure rehabilitation, background literat
+Through this initiative and other SERN/New Graph led initiatives, the Provincial Fish Passage Remediation Program and connectivity restoration planning **by the Canadian Wildlife Federation** [@mazany-wright_etal2021BulkleyRiver], `bcfishpass` has been designed to prioritize potential fish passage barriers for assessment or remediation. The software is under continual development and has been designed and constructed by @norris2021smnorrisbcfishpass using sql and python based shell script libraries to generate a simple model of aquatic habitat connectivity. The model identifies natural barriers (ex. steep gradients for extended distances) and hydroelectric dams to classifying the accessibility upstream by fish [@norris2021smnorrisbcfishpass]. On potentially accessible streams, scripts identify known barriers (ex. waterfalls >5m high) and additional anthropogenic features which are primarily road/railway stream crossings (i.e. culverts) that are potentially barriers. To prioritize these features for assessment or remediation, scripts report on how much modelled potentially accessible aquatic habitat the barriers may obstruct. The model can be refined with known fish observations upstream of identified barriers and for each crossing location, the area of lake and wetland habitat upstream, species documented upstream/downstream, an estimate of watershed area (on 2nd order and higher streams), mean annual precipitation weighted to upstream watershed area and channel width can be collated using `bcfishpass`, `fwapg` and `bcfishobs`.  This, information, can be used to provides an indication of the potential quantity and quality of habitat potentially gained should fish passage be restored by comparing to user defined thresholds for the aforementioned parameters. A discussion of the methodology to derive channel width is below.
```

Comment:
bcfishpass is the result of over 15 years of work that we have been involved in!  CWF jumped on board in 2020 so although it is great to have more dollars flowing into the program from their funders not sure we need to be putting them in bold here.

Reply:
Ok, I just did that because Betty said to add bolded text